build-test.xml

```diff
@@ -6549,6 +6551,44 @@ virtualHostname="localhost"
 		</if>
 	</target>
 
+	<target name="deploy-commerce-license-li">
+		<local name="start.date" />
+
+		<beanshell>
+			<![CDATA[
+				long startDate = new Date().getTime();
+				project.setProperty("start.date", Long.toString(startDate));
+			]]>
+		</beanshell>
+
+		<if>
+			<and>
+				<isset property="commerce.license.owner" />
+				<isset property="osb.api.token" />
+			</and>
+			<then>
+				<exec executable="curl" outputproperty="commerce.license">
+					<arg value="https://customer-uat.liferay.com/api/jsonws/osb-portlet.licensekey/generate-commerce-license-key?owner=${commerce.license.owner}&amp;startDate=${start.date}&amp;licenseLifetime=2592000000" />
```

Comment:
@xbrianlee is this URL safe to put in the public code? I feel like it should be stored privately, e.g. in the jenkins-ee code.

Reply:
@jpince It only works within our network and only if you pass in valid token.

Yeah, his code should go where our portal licensing code is for parity. I can ask CI Infra to help with this.